portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,114 @@
 <html>
```

Comment:
This deserves a special shoutout for being one of the best designed sites I've seen as a PA in SPS!
 👏 🎈 🥇